challenge_2/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="container">
+    <div class="boxInside"></div>
+    <div class="bigCircle"></div>
+    <div class="smallCircle"></div>
+  </div>
+</body>
+</html>
```

Comment:
Remember to add a blank line at the end of the file.
```suggestion
</html>

```